api/v1beta1/flinkcluster_types.go

```diff
@@ -391,6 +394,9 @@ type FlinkClusterSpec struct {
 
 	// Config for GCP.
 	GCPConfig *GCPConfig `json:"gcpConfig,omitempty"`
+
+	// The maximum number of revision history to keep.
+	RevisionHistoryLimit *int32 `json:"revisionHistoryLimit,omitempty"`
```

Comment:
What's the default value? Also need to add it to the CRD doc.

Reply:
Done.